driver_test.go

```diff
@@ -12,26 +12,246 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package spannerdriver_test
+package spannerdriver
 
 import (
+	"cloud.google.com/go/spanner"
+	"context"
 	"database/sql"
+	"fmt"
 	"log"
+	"os"
+	"reflect"
+	"runtime/debug"
+	"testing"
 
-	spannerdriver "github.com/rakyll/go-sql-driver-spanner"
```

Comment:
You need to import this as the side effect allows you to connect to spanner. `gofmt` removed it because the alias `spannerdriver` was unused. See [here](https://golang.org/doc/effective_go.html#blank_import) for how to import for side effects.

Reply:
I actually removed that as it was importing a a version of the driver that still has the byte read bug, if run with 'go test .' from the dir containing the package files it runs the init function inside the driver.go file which registers itself & reads bytes ok 🙂

Reply:
You're absolutely right! The package is the same so the driver is already imported. You may disregard the comment although please still read about importing for side effects for future reference :)

Reply:
Will do!